app/code/Magento/InventoryInStorePickupFrontend/Model/Validator/StorePickUpValidator.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickupFrontend\Model\Validator;
+
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\Validation\ValidationResultFactory;
+use Magento\InventoryInStorePickupApi\Api\GetPickupLocationsInterface;
+use Magento\InventoryInStorePickupApi\Model\SearchRequestBuilderInterface;
+use Magento\Store\Model\ScopeInterface;
+
+/**
+ * Verify, if store pick-up available for given website.
+ */
+class StorePickUpValidator
```

Comment:
What exactly this class validates? Could you please change to the name more explicit?

Reply:
Renamed